jenkins_jobs/trigger_weekly_host_os_build/pre_build_script.sh

```diff
@@ -53,5 +53,6 @@ write_comment "$BUILD_ISO_TRIGGER_PHRASE"
 create_release_notes
 create_pull_request "${GITHUB_ORGANIZATION_NAME}.github.io"
 
+echo "${TRIGGERED_JOB_NUMBER_build_host_os}" > BUILD_JOB_NUMBER
```

Comment:
This script is executed before the build, so you do not have this number yet. You could use this variable directly in the `copy_artifacts.sh` scripts.